### PR TITLE
Pairs-with view #52

### LINKED ISSUE #52: Slides Block: Product overlay
Extends #34 

Clicking on product icon will open an overlay for that product.

image

image

### PR DESCRIPTION
**Pull request by atopper**
## Issue

Fix #52

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/drafts/top/slides-pairswith
- After: https://pairs_with_52--sgws-creative--hlxsites.hlx.page/drafts/top/slides-pairswith

- Landing Page: https://pairs_with_52--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1-top-draft

Original: https://sgcreative.southernglazers.com/bloominbrands/p/1

Author Example: https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B1098E2C0-145B-49CF-AB3B-858225FE4AB7%7D&file=slides_pairswith.docx&action=default&mobileredirect=true
Adding a header, paragraph and picture will set up the pairs-with view.

## 📝 Description:
Allow the user to author a "pairs-with" view in the slides.

### CODE REVIEW

File: blocks/slides/slides.css [hunk 1]
```diff
@@ -148,8 +304,8 @@
 }
 
 /* animations */
-@media (prefers-reduced-motion: no-preference) {
```
Comment (atopper) [inline blocks/slides/slides.css:151]: This style made the buttons go opaque for me (bad default with no theme?).  I'm not sure 'reduce' is good either though.

**Review by tmathern — CHANGES_REQUESTED** [review]
2 things: 
- Could you create a draft of the landing page in which this is used?
- It would be nice if the "pairs with" has the same height as the underlying slide, so nobody becomes sea-sick. (This may need js magic).

File: blocks/slides/slides.css (hunk 1 above)
Comment (tmathern) [reply blocks/slides/slides.css:151]: ACK. On which browser?

Comment (atopper) [reply blocks/slides/slides.css:151]: Windows, Chrome.  If the user hasn't specified that motion should be reduced, the value becomes "no-preference".   `Reduce` means to keep it to a minimum.

**Comment by atopper** [comment]
> It would be nice if the "pairs with" has the same height as the underlying slide, so nobody becomes sea-sick. (This may need js magic).

This makes sense visually, but what if the 'pairs-with' text is much larger than the text in the first view?  Do I make the origin view a lot taller, which would look weird with the empty space?  Or do I limit the 'pairs with' view, and add scroll bars, which would look weird in this presentation model?  Or do we lock the height, and assume their text will be authored properly to fit nicely (which it seems like they have)?

**Comment by atopper** [comment]
@tmathern  I'm trying to make a landing page with my changes, but here's my branch using the original /p/1 doc
https://pairs_with_52--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1

I'm having trouble copying the 1.docx to my draft.  I believe some of the links, themes, etc. are all lost.

**Review by tmathern — APPROVED** [review]

**Pull request merged**